### PR TITLE
[pipelines] Kubeflow Pipelines upgrade to 1.0.1

### PR DESCRIPTION
**Pull request by quaark**
### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (e.g. `[mychartname]`)

### CODE REVIEW

File: stable/pipelines/templates/metadata/grpc-configmap.yaml [hunk 1]
```diff
@@ -4,9 +4,9 @@ kind: ConfigMap
 metadata:
   name: metadata-grpc-configmap
   labels:
-    component: metadata-server
+    component: metadata-grpc-server
 {{ include "pipelines.commonLabels" . | indent 4 }}
 data:
-  METADATA_GRPC_SERVICE_HOST: "metadata-service"
+  METADATA_GRPC_SERVICE_HOST: "metadata-grpc-server"
```
Comment (omesser) [inline stable/pipelines/templates/metadata/grpc-configmap.yaml]: The service name is now `metadata-grpc-service` this looks like it needs to be the same as https://github.com/v3io/helm-charts/pull/488/files#diff-d9fc3ec5e5971729df4efe8261f34cc8R5

File: stable/pipelines/templates/metadata/grpc-deployment.yaml [hunk 2]
```diff
@@ -2,21 +2,21 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: metadata
+  name: metadata-grpc-deployment
```
Comment (omesser) [inline stable/pipelines/templates/metadata/grpc-deployment.yaml:5]: is the `-deployment` suffix to comply with kubeflow/pipelines naming or your doing?
If to comply 👍 
else - I think we can let this go (putting the resource type in the name here, it's sometimes done but IIRC on deployments it's not)

File: stable/pipelines/templates/ml-pipeline/apiserver/deployment.yaml [hunk 3]
```diff
@@ -21,6 +21,8 @@ spec:
       containers:
       - name: ml-pipeline-api-server
         env:
+#        - name: OBJECTSTORECONFIG_SECURE
+#          value: "false"
```
Comment (omesser) [inline stable/pipelines/templates/ml-pipeline/apiserver/deployment.yaml]: 🤔 this seems important (I dont remember what it does though)

File: stable/pipelines/templates/ml-pipeline/ui/configmap.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    component: ml-pipeline-ui
+{{ include "pipelines.commonLabels" . | indent 4 }}
+data:
+   viewer-pod-template.json:  |-
+    {
+        "spec": {
+            "serviceAccountName": "kubeflow-pipelines-viewer"
+        }
+    }
+{{- end }}
```
Comment (omesser) [inline stable/pipelines/templates/ml-pipeline/ui/configmap.yaml]: newline eof

**Review by omesser — CHANGES_REQUESTED** [review]
<slow clap>

File: stable/pipelines/templates/metadata/grpc-deployment.yaml (hunk 2 above)
Comment (quaark) [reply stable/pipelines/templates/metadata/grpc-deployment.yaml:5]: it's to comply with what they did 😄

File: stable/pipelines/templates/ml-pipeline/apiserver/deployment.yaml (hunk 3 above)
Comment (quaark) [reply stable/pipelines/templates/ml-pipeline/apiserver/deployment.yaml]: Yeah I saw it was added but didn't want to add it - cos secure = false... WDYT?

File: stable/pipelines/templates/metadata/grpc-configmap.yaml (hunk 1 above)
Comment (quaark) [reply stable/pipelines/templates/metadata/grpc-configmap.yaml]: You are correct, it should be service. I'm checking to see everything works and then I'll update the PR

File: stable/pipelines/templates/ml-pipeline/apiserver/deployment.yaml (hunk 3 above)
Comment (omesser) [reply stable/pipelines/templates/ml-pipeline/apiserver/deployment.yaml]: question is if we need it or not, not how it looks in the chart 🤷

**Review by omesser — APPROVED** [review]

**Pull request merged**